src/CoreFoundation/DispatchData.cs

```diff
@@ -88,6 +88,18 @@ public static DispatchData FromBuffer (IntPtr buffer, nuint size)
 			return new DispatchData (dd, owns: true);
 		}
 
+		// create a dd using the data of the span 
+		public static DispatchData FromReadOnlySpan (ReadOnlySpan<byte> content)
+		{
+			unsafe {
+				fixed (byte* ptr = content) {
+					// set the block that will be used to clean the data once we are done with it
+					var dd = dispatch_data_create ((IntPtr) ptr, (nuint) content.Length, IntPtr.Zero, destructor: IntPtr.Zero);
```

Comment:
Could you explain the use of `IntPtr.Zero` for the queue and destructor parameters?

Reply:
The info is in the comment from @rolfbjarne https://github.com/xamarin/xamarin-macios/pull/7690#discussion_r364950115